aggregator/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+name = "aggregator"
```

Comment:
Should this be another binary in the call-home package?

Reply:
IMHO, i dont find it a good fit in callhome  as we can have the stats even when callhome is disabled.

Reply:
But isn't the aggregator just for call-home?

Reply:
Only if it's appropriate, which is why I'm asking, is the aggregator specific for call home? @datacore-vvarakantham @chriswldenyer @niladrih

Reply:
The aggregator should be specific to call-home. Is there a use case for it without call-home? I didn't think there was.

Reply:
Just wanted to double check, in such case should be a binary part of call home.